src/frontend/screens/Settings/components/EnableDXVKFpsLimit.tsx

```diff
@@ -0,0 +1,57 @@
+import React, { useContext } from 'react'
+import { useTranslation } from 'react-i18next'
+import { ToggleSwitch, TextInputField } from 'frontend/components/UI'
+import useSetting from 'frontend/hooks/useSetting'
+import ContextProvider from 'frontend/state/ContextProvider'
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import { faCircleInfo } from '@fortawesome/free-solid-svg-icons'
+import SettingsContext from '../SettingsContext'
+
+const EnableDXVKFpsLimit = () => {
+  const { t } = useTranslation()
+  const { platform } = useContext(ContextProvider)
+  const { isLinuxNative, isMacNative } = useContext(SettingsContext)
+  const isWin = platform === 'win32'
+  const [enableDXVKFpsLimit, setDXVKFpsLimit] = useSetting(
+    'enableDXVKFpsLimit',
+    false
+  )
+  const [DXVKFpsCap, setDXVKFpsCap] = useSetting('DXVKFpsCap', '')
+
+  if (isWin || isLinuxNative || isMacNative) {
```

Comment:
This should do it. I believe this results the same `!shouldRenderFpsOption` used for the ShowFPS.

Reply:
yes, that should do it